unittests/pythontests.py

```diff
@@ -64,7 +64,12 @@ def _test_bytecompile(self, py2=False):
             for file in files:
                 realfile = os.path.join(root, file)
                 if file.endswith('.py'):
-                    cached = glob.glob(realfile+'?') + glob.glob(os.path.join(root, '__pycache__', os.path.splitext(file)[0] + '*.pyc'))
+                    # FIXME: relpath must be adjusted for windows path behaviour
+                    try:
+                        root = os.path.join(sys.pycache_prefix, os.path.relpath(root, '/'))
+                    except (TypeError, AttributeError):
```

Comment:
nitpick: this can be `if hasattr(sys, "pycache_prefix"):` instead of a `try..except`. The exceptions listed can be emitted due to other reasons than that attribute not existing, such as due to typos.

Reply:
Thanks, I couldn't remember the exact syntax (and I kept coming up with the Lua version, for some reason).